pkg/registry/servicecatalog/rest/storage_servicecatalog.go

```diff
@@ -56,7 +56,7 @@ func (p StorageProvider) NewRESTStorage(
 		return nil, err
 	}
 
-	apiGroupInfo := genericapiserver.NewDefaultAPIGroupInfo(servicecatalog.GroupName, api.Registry, server.Scheme, server.ParameterCodec, server.Codecs)
+	apiGroupInfo := genericapiserver.NewDefaultAPIGroupInfo(servicecatalog.GroupName, api.Registry, api.Scheme, api.ParameterCodec, api.Codecs)
```

Comment:
need to check that this was removed from server or moved to api or see what happened.

Reply:
@MHBauer yeah it was.